### DISCUSSION TITLE
[explore]Improve Experience of adding chart to NEW dashboard from Explore

### DISCUSSION THREAD

**Discussion by zuzana-vej** [apache/superset#19210] [Ideas]
**Is your feature request related to a problem? Please describe.**

The current experience of adding chart from Explore to a NEW dashboard is confusing for some users. 

![Screen Shot 2021-01-25 at 7 38 55 PM](https://user-images.githubusercontent.com/61221714/105797622-33b7d580-5f45-11eb-9f86-e47dea56fbd7.png)

In the textbox, it says "Select a dashboard OR create a new one" but it's not apparent to users that all they need to do is type a new dashboard name, press enter, and then the "Save & go to dashboard" button will be enabled. User thought this was a bug. 

**Describe the solution you'd like**
In the past, there used to be a "Create new dashboard option" from the dropdown and it opened up a dialog box to type a dashboard name - also slightly clunky. Maybe if the text said something like "Type new dashboard name here or select from existing". There just needs to be some signal that it's type-able.

**Describe alternatives you've considered**
Few options:
1. easiest: Better text in the text field - for example _"Type new dashboard name here or select from existing"_
2. button create new dashboard, which opens a popup where user fills new dashboar details (apparently past solution)
3. tooltip, letting user know to "Type new dashboard name and click enter"

**Comment by junlincc** [comment]
> it's not apparent to users that all they need to do is type a new dashboard name, press enter, and then the "Save & go to dashboard" button will be enabled.

suffering from the same confusion here 😞 we may have to go with the easiest solution for now. tag `design-sys-revisit`


@mihir174

**Comment by ktmud** [comment]
Related #10355

**Comment by mihir174** [comment]
Here's a common pattern for saving to a new instance:
<img width="1972" alt="Slice 1 (6)" src="https://user-images.githubusercontent.com/64227069/106075490-92568e00-60c2-11eb-87c0-c7db4bd800db.png">
